src/main/java/com/prgrms/board/dto/post/PostCreateRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.prgrms.board.dto.post;
+
+import org.hibernate.validator.constraints.Length;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Positive;
+
+public record PostCreateRequest(
+        @Positive Long userId,
+        @NotBlank(message = "제목이 비어있습니다.") @Length(max = 250) String title,
+        String content) {
+}
```

Comment:
```suggestion
public record PostCreateRequest(
        @Positive 
        Long userId,
        
        @NotBlank(message = "제목이 비어있습니다.") @Length(max = 250) 
        String title,
        
        String content
        ) {
}
```

레코드 어떻게 쓰는지 잘모르지만, 일단 포메팅을 한번 바꿔봤습니다! 저렇게해도 돌아가는지 궁금하네요 ㅋㅋㅋ